etcddb/etcddb_client.go

```diff
@@ -81,6 +81,28 @@ func (client *EtcdClient) Get(key string) (value string, ok bool, err error) {
 	return
 }
 
+// GetWithRange gets all values where key is prefix
+func (client *EtcdClient) GetWithRange(key string) (values []string, ok bool, err error) {
```

Comment:
I would propose renaming to GetByKeyPrefix

Reply:
Renamed the method to GetByKeyPrefix.